src/components/GithubLink/GithubLink.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Icon } from '../Icon/Icon';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { CSSTransition } from 'react-transition-group';
+
+import './GithubLink.scss';
+
+const GithubLink = ({ name, color, size, className }) => {
+  const [isShown, setIsShown] = useState(false);
+  const setTooltipVisibility = (boolean) => {
```

Comment:
I recommend changing the param name from `boolean` to something like `isVisible`